frontend/src/components/Analysis/utils.tsx

```diff
@@ -211,7 +211,9 @@ export function parseFineGrainedResults(
       );
 
       const analysisName = myResult.name;
-      const analysisDescription = myAnalysis?.description;
+      const analysisDescription = myAnalysis?.description
```

Comment:
nit: I think this can be written as `myAnalysis?.description && analysisName`.

Reply:
Thanks!